### ISSUE TITLE
Context appears then disappears upon consecutive runs of `ttag update`

### ISSUE THREAD

**Issue by forkerino** [ttag-org/ttag-cli#84]
I have a .po file with the following:

```
#: app/grids/grids.utils.js:440
msgid "NE"
msgstr "NO"
```
I've added context to it. When I do `ttag update`, I first get: 
```
#: app/grids/grids.utils.js:440
msgctxt "directions"
msgid "NE"
msgstr ""
```
Note the translation is gone, but the context is present. No problem, I'll add the translation back in. 

Now I have more strings to translate and run `ttag update` again, resulting in: 

```
#: app/grids/grids.utils.js:440
msgid "NE"
msgstr "NO"
```
(No context).

When I run it again, I get the second result again. So context gets added, then removed, then added, then removed, etc. I believe the context is not passed properly somewhere in the update function.

**Comment by hitbear518** [comment]
Same issue here

**Comment by AlexMost** [comment]
Hey, thanks for the report! Will take a look

**Comment by dimaqq** [comment]
Here's a sample diff:
```diff
> diff -u 1.po 2.po
--- 1.po	2019-11-21 15:08:04.000000000 +0900
+++ 2.po	2019-11-21 15:08:15.000000000 +0900
@@ -512,74 +512,68 @@
 "and the administration service must be enabled in your access policy group"
 msgstr ""

-#: src/user_table/index.js:65
-#. FIXME how can we `setPage(1)` ?
-msgid "Display Name"
-msgstr ""
-
-#: src/user_table/index.js:69
-#. FIXME how can we `setPage(1)` ?
-msgid "Policy Name"
-msgstr ""
-
 #: src/user_table/index.js:64
 #. FIXME how can we `setPage(1)` ?
-msgctxt "user table header"
 msgid "Username"
 msgstr ""

 #: src/user_table/index.js:66
 #. FIXME how can we `setPage(1)` ?
-msgctxt "user table header"
 msgid "Office365 ID (UPN)"
 msgstr ""

 #: src/user_table/index.js:67
 #. FIXME how can we `setPage(1)` ?
-msgctxt "user table header"
 msgid "Status"
 msgstr ""

 #: src/user_table/index.js:68
 #. FIXME how can we `setPage(1)` ?
-msgctxt "user table header"
 msgid "Last Login (UTC)"
 msgstr ""

+#: src/user_table/index.js:65
+#. FIXME how can we `setPage(1)` ?
+msgctxt "user table header"
+msgid "Display Name"
+msgstr ""
+
+#: src/user_table/index.js:69
+#. FIXME how can we `setPage(1)` ?
+msgctxt "user table header"
+msgid "Policy Name"
+msgstr ""
+
 #: src/user_table/index.js:99
+msgctxt "status"
 msgid "enabled"
 msgstr ""

 #: src/user_table/index.js:99
+msgctxt "status"
 msgid "disabled"
 msgstr ""

 #: src/user_table/index.js:112
-msgctxt "short"
 msgid "View..."
 msgstr ""

 #: src/user_table/index.js:113
-msgctxt "short"
 msgid "Edit..."
 msgstr ""

 #: src/user_table/index.js:114
-msgctxt "short"
 msgid "Change Password..."
 msgstr ""

 #: src/user_table/index.js:115
-msgctxt "short"
 msgid "Delete..."
 msgstr ""

 #: src/comp/SubmitBar.js:76
-msgctxt "short"
 msgid "Close"
 msgstr ""

 #: src/comp/Timestamp.js:46
-msgctxt "short"
 msgid "Toggle"
 msgstr ""
\ No newline at end of file
```

Weirdly there are exactly 2 outputs, and the result of `ttag update` keeps bouncing between the two 🤔

**Comment by MrOrz** [comment]
Hi all, I have composed a PR that should fix this issue, feel free to give it a try:
https://github.com/ttag-org/ttag-cli/pull/98

**Comment by AlexMost** [comment]
@MrOrz thanks for your help! Fix is already available in `ttag-cli@1.8.3`
